### PR TITLE
Updates post rename

### PR DESCRIPTION
**Pull request by rpavlik**
Add diagrams, revise FOV, small behavior changes. This is what I've been using regularly, and most of my later work is based on this branch.

### CODE REVIEW

File: server/src/ems/gst/ems_gstreamer_pipeline.c [hunk 1]
```diff
@@ -171,7 +171,7 @@ on_offer_created(GstPromise *promise, GstElement *webrtcbin)
 static void
 webrtc_on_data_channel_cb(GstElement *webrtcbin, GObject *data_channel, struct ems_gstreamer_pipeline *egp)
 {
-	U_LOG_E("called!!!!");
+	U_LOG_I("called!!!!");
```
Comment (fredinfinite23) [inline server/src/ems/gst/ems_gstreamer_pipeline.c]: We could have :
U_LOG_I("webrtc_on_data_channel_cb called");

File: server/src/ems/ems_hmd.cpp [hunk 2]
```diff
@@ -203,24 +203,22 @@ ems_hmd_create(ems_instance &emsi)
...
+	eh->base.hmd->distortion.fov[0] = (xrt_fov){
+	    // .angle_left = -0.75f,
+	    .angle_left = -0.855f,
+	    .angle_right = 0.785f,
+	    .angle_up = 0.838f,
+	    .angle_down = -0.873f,
+	};
+	eh->base.hmd->distortion.fov[1] = (xrt_fov){
+	    .angle_left = -0.785f,
+	    // .angle_right = 0.75f,
```
Comment (fredinfinite23) [inline server/src/ems/ems_hmd.cpp]: Specific reason for keeping this here commented ?

Comment (rpavlik) [reply server/src/ems/ems_hmd.cpp]: iirc this was the difference between two different Quest 2 units.

File: server/src/ems/gst/ems_gstreamer_pipeline.c (hunk 1 above)
Comment (rpavlik) [reply server/src/ems/gst/ems_gstreamer_pipeline.c]: not sure why this is still in here, tbh

File: server/src/ems/ems_hmd.cpp (hunk 2 above)
Comment (rpavlik) [reply server/src/ems/ems_hmd.cpp]: Removed.

File: server/src/ems/gst/ems_gstreamer_pipeline.c (hunk 1 above)
Comment (rpavlik) [reply server/src/ems/gst/ems_gstreamer_pipeline.c]: fixed.

**Comment by fredinfinite23** [comment]
All comments addressed, this looks good thx !

**Review by fredinfinite23 — APPROVED** [review]
SHIP IT !

**Pull request merged**